pkgs/applications/networking/xtreme-download-manager/default.nix

```diff
@@ -0,0 +1,62 @@
+{ pkgs
```

Comment:
please don't import pkgs, you should prefer the individual packages

Reply:
Fixed.